alpha/external-web/app/routes/about-you.js

```diff
@@ -2,6 +2,7 @@ var router = require('../routes')
 var client = require('../services/eligibility-client')
 var eligibilityFlag = require('../services/eligibility-flag')
 var logger = require('../services/bunyan-logger')
+var util = require('util')
```

Comment:
I don't think it's needed, it was used in the example on [express-validator github](https://github.com/ctavan/express-validator) but I think we can use JSON.stringify instead